recipes/mne-python_1.5/construct.yaml

```
@@ -82,12 +82,14 @@ specs:
   - pyriemann =0.5
   - pyprep =0.4.2
   - openmeeg =2.5.6
+  # BLAS
+  # - openblas =0.3.23
```

I think I'd rather keep this, it should help speed up resolution, and we can rely on `outdated` to tell us to update
```suggestion
  - openblas =0.3.23
```

---

Fair enough!